src/ApiCore/OperationResponse.php

```diff
@@ -133,40 +164,68 @@ public function getName()
      * Poll the server in a loop until the operation is complete.
      *
      * Return true if the operation completed, otherwise return false. If the
-     * $options['maxPollingDuration'] setting is not set (or set <= 0.0) then
+     * $options['totalPollTimeoutMillis'] setting is not set (or set <= 0.0) then
      * pollUntilComplete will continue polling until the operation completes,
      * and therefore will always return true.
      *
      * @param array $options {
      *                       Options for configuring the polling behaviour.
      *
-     *     @type float $pollingIntervalSeconds The polling interval to use, in seconds.
-     *                                          Default: 1.0
-     *     @type float $maxPollingDurationSeconds The maximum amount of time to continue polling.
-     *                                            Default: 0.0 (no maximum)
+     *     @type float $initialPollDelayMillis The initial polling interval to use, in seconds.
```

Comment:
```diff
+ in milliseconds
- in seconds
```

(applies to the other properties too, as well as in the constructor)

Reply:
Fixed